src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.tsx

```diff
@@ -77,6 +76,14 @@ export const ContactFlowColumn: React.FC<Props> = ({
     skip: !accountListId || !appealStatus,
   });
 
+  useEffect(() => {
```

Comment:
Instead of exposing a boolean with context, why don't we expose the `refetch` function and let the child components call `refetch` themselves? That will also avoid two extra renders to set the state to `true` and then back to `false`.

Reply:
Sorry didn't realize that `refetch` is in a child component, not the provider component.